db/base/models.py

```diff
@@ -11,6 +11,8 @@
 from db.base.helpers import gridsquare
 
 DATA_SOURCES = ['manual', 'network', 'sids']
+SATELLITE_STATUS = ['Alive', 'Dead', 'Re-entered']
+SATELLITE_STATUS_CODES = ['a', 'd', 'r']
```

Comment:
Why use two separete dicts? If you want to have a short code to use on template later for css you can do that with one dict and don't use `zip` on the model. See [this example](https://docs.djangoproject.com/en/1.11/ref/models/fields/#choices) from the documentation.